test/language/generators/generator.declareMethod.shadow-parameter-catch.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+ GeneratorMethod early SyntaxError when lexical declaration
+ inside generator shadows parameter name 
+es6id: 14.4.1
+author: Sam Mikes
+description: GeneratorMethod error with lexical shadowing
+negative: SyntaxError
+---*/
+
+var obj = {
+    *foo(a) {
+        try {
+        }
+        catch (e) {
+            let a = 3;
```

Comment:
I think if you rename let a = 3 to let e = 3 it should error, but this is not a generator test but a try/catch test IMO. Fine with removal :)